draft-ietf-teas-5g-ns-ip-mpls.md

```diff
@@ -171,9 +171,10 @@ This document describes a Network Slice realization model for IP/MPLS networks w
 
 #  Introduction
 
-This document focuses on network slicing for 5G networks, covering the connectivity between network functions across multiple domains such as edge clouds, data centers, and the WAN.  The document describes a Network Slice realization model that fulfills 5G slicing requirements by using existing IP/MPLS technologies to optimally control Service Level Agreements (SLAs) offered for 5G slices.
+This document focuses on network slicing for 5G networks, covering the connectivity between network functions across multiple domains such as edge clouds, data centers, and the WAN.  The document describes a Network Slice realization approach that fulfills 5G slicing requirements by using existing IP/MPLS technologies to optimally control Service Level Agreements (SLAs) offered for 5G slices.
 
-This work is compatible with {{!I-D.ietf-teas-ietf-network-slices}} which defines a generic framework for network slicing built using IETF technologies. The  framework introduces the concept of a Network Resource Partition (NRP), which is a collection of resources identified in the underlay network. This framework is best suited for the WAN (provider network), using just one NRP or multiple NRPs.
+This work is compatible with {{!I-D.ietf-teas-ietf-network-slices}} which describes network slicing in the context of networks built from IETF technologies. This framework introduces the concept of a Network Resource Partition (NRP), which is a collection of resources identified in the underlay network.
```

Comment:
```suggestion
This work is compatible with the framework described in {{!I-D.ietf-teas-ietf-network-slices}} which describes network slicing in the context of networks built from IETF technologies. This framework introduces the concept of a Network Resource Partition (NRP), which is a collection of resources identified in the underlay network.
```

otherwise "This framework.." won't be OK in the next sentence.